index.html

```diff
@@ -0,0 +1,449 @@
+              <a
+                class="nav-link menu-item px-3"
+                aria-current="page"
+                href="#about-me-anchor"
+                id="aboutMeText"
+                ><span>01. </span>about me</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link menu-item px-3" href="#my-work-anchor" id="myWorkText"
+                ><span>02. </span>my work</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link menu-item pe-5 ps-3" href="#my-contact-anchor" id="myContactText"
+                ><span>03. </span>my contact</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <!-- add the scrollspy to highlight the section in the menu as we scroll -->
+    <div
+        data-bs-spy="scroll"
+        data-bs-target="#navbar-scrollspy"
+        data-bs-offset="0"
+        class="scrollspy-example"
+        tabindex="0"
+        id="mainSection"
+        >
+        <!-- ABOUT ME -->
+        <div class="about-me" id="about-me-anchor">
+          <!-- about me title section on the left -->
+          <div class="about-me-title">
+            <h2>about me</h2>
+          </div>
+
+          <!-- text with info and intro about myself / this is regular text -->
+          <div class="about-me-text">
+            <h1>
+              <b>Hi!</b>
+            </h1>
+            <h2>Welcome to my portfolio page</h2>
+            <p>
+              My name is Nabil. I was born in Honduras and I currently live in
+              "poor-but-sexy" Berlin.
+            </p>
+          </div>
+        </div>
+
+        <!-- MY WORK -->
+        <div class="my-work" id="my-work-anchor">
+          <!-- my work title section on the left -->
+          <div class="my-work-title">
+            <h2>my work</h2>
+          </div>
+
+          <!-- this is where all projects are configured and then located on the right side -->
+          <div class="projects">
+            <!-- Project #1: The Boring App -->
+            <div class="project mb-3" id="project-1">
+              <a href="https://github.com/nnassarv/TheBoringApp">
+                <div
+                  id="carousel-boringApp"
+                  class="carousel carousel-dark slide carousel-fade"
+                  data-bs-ride="carousel"
+                >
+                  <div class="carousel-inner">
+                    <div class="carousel-item active">
+                      <img
+                        src="./assets/images/boring-app-welcome-page.png"
+                        class="d-block w-100"
+                        alt="..."
+                      />
+                    </div>
+                    <div class="carousel-item">
+                      <img
+                        src="./assets/images/make-a-choice.jpeg"
+                        class="d-block w-100"
+                        alt="..."
```

Comment:
if you can, add alt texts to all of your img tags as this really helps to make your site more accessible to users who may struggle with their sight.
Not only that, if somebody is experiencing internet issues and the images from your portfolio do not manage load to the screen as a result the alt text will display in place and help the user to know what image should've been there.

Reply:
i need to keep a post-it note about alt texts... i always forget them! Thanks Dom